docs/migrating-from-other-versions.md

```diff
@@ -0,0 +1,69 @@
+# Migrating From Other Versions
```

Comment:
does the different npm namespace and repository affect anyone who is upgrading?

Reply:
Slightly, but it's all NPM common knowledge.  They will install it via the new name, and when its used, it will use the new name.  But all of that is in the README (as `@logdna/logger`)